models/artifacts.go

```diff
@@ -0,0 +1,24 @@
+package models
+
+import (
+	"time"
+
+	"github.com/google/uuid"
+)
+
+// Artifact represents the artifacts table
+type Artifact struct {
+	ID            uuid.UUID  `json:"id"`
+	CheckID       uuid.UUID  `json:"check_id,omitempty"`
+	CheckTime     time.Time  `json:"check_time,omitempty" time_format:"postgres_timestamp"`
+	PlaybookRunID uuid.UUID  `json:"playbook_run_id,omitempty"`
+	ConnectionID  uuid.UUID  `json:"connection_id,omitempty"`
+	FileID        string     `json:"file_id"`
```

Comment:
What is the difference between this and id ?

Reply:
_FileID_ would be the id of the file on the store and id is just the database row id.
eg: For s3, it would be the object id